src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -1385,7 +1672,11 @@ class SpliceTestsCommon : LightningTestSuite() {
 
             // Alice's transactions confirm.
             val (alice4, actionsAlice4) = alice3.process(ChannelCommand.WatchReceived(WatchEventConfirmed(alice3.channelId, BITCOIN_TX_CONFIRMED(bobCommitTx), alice3.currentBlockHeight, 43, bobCommitTx)))
-            assertEquals(actionsAlice4.size, 1)
+            val failedOutgoingHtlcs = when (bobCommitTx.txid == alice3.commitments.latest.remoteCommit.txid) {
```

Comment:
Same here, it's better to check that we have the right actions for pending HTLCs. That allowed me to find a bug, which is fixed in https://github.com/ACINQ/lightning-kmp/commit/b7b436466d9b3e0d1ca962ab4794f74cc41aedeb